modules/nodes-containers-dev-fuse-configuring.adoc

```diff
@@ -4,63 +4,136 @@
 
 .Procedure
 
-. Create the pod.
+. Define the pod with `/dev/fuse` access:
 +
-[source,terminal]
+Create a YAML file named `fuse-builder-pod.yaml` with the following content: 
++
+[source,yaml]
 ----
-$ oc exec -ti no-priv -- /bin/bash
+apiVersion: v1
+kind: Pod
+metadata:
+  name: fuse-builder-pod
+  annotations:
+    io.kubernetes.cri-o.Devices: "/dev/fuse" <1>
+spec:
+  containers:
+  - name: build-container
+    image: quay.io/podman/stable <2>
+    command: ["/bin/sh", "-c"]
+    args: ["echo 'Container is running. Use oc exec to get a shell.'; sleep infinity"] <3>
+    securityContext: <4>
+      runAsUser: 1000
 ----
 +
+<1> The `io.kubernetes.cri-o.Devices: "/dev/fuse"` annotation makes the FUSE device available.
+<2> This annotation specifies a container that uses an image that includes `podman` (for example, `quay.io/podman/stable`).
+<3> This command keeps the container running so you can `exec` into it.
+<4> This annotation specifies a `securityContext` that runs the container as an unprivileged user (for example, `runAsUser: 1000`).
+* 
++
+[NOTE]
+====
+Depending on your cluster's Security Context Constraints (SCCs) or other policies, you might need to further adjust the `securityContext` specification, for example, by allowing specific capabilities if `/dev/fuse` alone is not sufficient for `fuse-overlayfs` to operate.
+====
+
+. Create the pod:
++
+Apply the YAML file to create the pod in your OpenShift project:
++
 [source,terminal]
 ----
-$ cat >> Dockerfile <<EOF
-FROM registry.access.redhat.com/ubi9
-EOF
+$ oc apply -f fuse-builder-pod.yaml
 ----
+
+. Verify that the pod is running:
 +
 [source,terminal]
 ----
-$ podman build .
+$ oc get pods fuse-builder-pod
 ----
 
-. Implement `/dev/fuse` by adding the `/dev/fuse` device to the `io.kubernetes.cri-o.Devices` annotation.
+. Access the pod and prepare the build environment:
 +
-[source,yaml]
+Once the `fuse-builder-pod` pod is in the `Running` state, open a shell session into the `build-container` environment:
++
+[source,terminal]
 ----
-io.kubernetes.cri-o.Devices: "/dev/fuse"
+$ oc exec -ti fuse-builder-pod -- /bin/bash
 ----
 +
-For example:
+You are now inside the container. Because the default working directory might not be writable by the unprivileged user, change to a writable directory like `/tmp`:
 +
-[source,yaml]
+[source,terminal]
 ----
-apiVersion: v1
-kind: Pod
-metadata:
-  name: podman-pod
-  annotations:
-    io.kubernetes.cri-o.Devices: "/dev/fuse"
+$ cd /tmp
+$ pwd
+/tmp
 ----
 
-. Configure the `/dev/fuse` device in your pod specifications.
+. Create a dockerfile and build an image using Podman:
 +
-[source,yaml]
+Inside the pod's shell and within the `/tmp` directory, you can now create a `Dockerfile` and use `podman` to build a container image. If `fuse-overlayfs` is the default or configured storage driver, Podman will be able to leverage `fuse-overlayfs` because of the available `/dev/fuse` device.
```

Comment:
```suggestion
Inside the pod's shell and within the `/tmp` directory, you can now create a `Dockerfile` and use `podman` to build a container image. If `fuse-overlayfs` is the default or configured storage driver, Podman is able to leverage `fuse-overlayfs` because of the available `/dev/fuse` device.
```